app/frontend/styles/_footer.scss

```diff
@@ -0,0 +1,13 @@
+.govuk-footer--app {
```

Comment:
Is `.app-govuk-footer` closer to what this is, an override in our app for the default footer styling?

Reply:
I went with this given BEM convention and that we are modifying an existing component, rather than creating an entirely new one.

Reply:
👌

Reply:
I went with this given BEM convention and that we are modifying an existing component, rather than creating an entirely new one.